.github/actions/pre-commit/action.yml

```
@@ -13,14 +13,22 @@ inputs:
     description: The python version of the local runner
     required: false
     default: "3.9"
+  auto-commit:
+    description: "enables auto-commit of eventual fixups (requires `permissions.contents: write` on the calling job)"
```

```suggestion
    description: >-
      Enables auto-commit of eventual fixups
      (requires `permissions.contents: write` on the calling job)
```

and nitpicking: this input would be better placed before the "python-version" input

Also, since I do have the behavior on some workflows :sweat_smile: afaik only the first fixups are done and all hooks after are not run, so the pre-commit command would need to be executed multiple times to get all the potential fixups... (I considered adding an integer to control how many times it should be run and gave up: called this action multiple times continuing on error instead)

---

I was actually looking for such an option and did not find the "fail-fast" one, thanks!
Now if it's disabled by default, it's probably not going to fix my issue, but I'll look into it :+1:

---

Actually the fail fast option was activated... perfect!